pod.go

```diff
@@ -872,6 +876,11 @@ func (p *Pod) start() error {
 		return err
 	}
 
+	// Before starting the container, POD's resources MUST BE updated
+	if err := p.updateResources(); err != nil {
```

Comment:
Based on my previous explanation, this can be removed from there. (moved to `createContainer()` in `container.go`)